### PR TITLE
Fix typos, grammar mistakes and ShellSession blocks in Helm Readme

### PR DESCRIPTION
**Pull request by dcermak**

### CODE REVIEW

File: src/bci_build/package/helm/README.md.j2 [hunk 1]
```diff
@@ -10,12 +10,16 @@
...
-To run Helm, use the following comamand:
+To run Helm, use the following command:
 
 ```ShellSession
-# podman run --rm -it  {{ image.pretty_reference }} <helm-sub-command>
+$ podman run --rm -it  {{ image.pretty_reference }} <helm-sub-command>
+```
 
-podman run --rm -it {{ image.pretty_reference }} version
+For instance, to display the helm version, run:
```
Comment (alexandrevicenzi) [inline src/bci_build/package/helm/README.md.j2:19]: **helm** should be **Helm** here I believe.

File: src/bci_build/package/helm/README.md.j2 [hunk 2]
```diff
@@ -24,10 +28,10 @@ For a comprehensive guide on getting started with Helm, refer to the [official H
 
 ### Connecting Helm container to the Host's Kubernetes Cluster
 
-To interact with a Kubernetes cluster running on the host, mount the Kubernetes configuration file (`kubeconfig`) into the container (use `--net=host` flag to allow container to use host’s network):
+To interact with a Kubernetes cluster running on the host, mount the Kubernetes configuration file (`kubeconfig`) into the container (use the `--net=host` flag to allow the container to use the host’s network):
```
Comment (alexandrevicenzi) [inline src/bci_build/package/helm/README.md.j2:31]: In a production scenario, you will never execute Helm on the host (aka control plane), always remote (dev machine).

The kubeconfig is not from the host cluster, it is from a dev machine.

> To interact with a Kubernetes cluster, mount the Kubernetes configuration file (`kubeconfig`) from the container host into the container (use the `--net=host` flag to allow the container to use the host’s network):

**Review by dirkmueller — APPROVED** [review]

File: src/bci_build/package/helm/README.md.j2 (hunk 1 above)
Comment (dirkmueller) [inline src/bci_build/package/helm/README.md.j2:19]:
```suggestion
For instance, to display the Helm version, run:
```

File: src/bci_build/package/helm/README.md.j2 (hunk 2 above)
Comment (dirkmueller) [reply src/bci_build/package/helm/README.md.j2:31]: I like the rewording suggestion, it makes it a lot clearer. There is a 2nd use case, to have a admin privileged pod running within the kubernetes cluster. this was the original request for this container. see e.g.

https://serverfault.com/questions/1013420/kubernetes-kubeconfig-inside-pod
https://stackoverflow.com/questions/72874454/how-to-access-kubeconfig-file-inside-containers

and similar. fact is we could be supporting both usecases here.

**Review by lvicoun — APPROVED** [review]
LGTM!

**Review by dirkmueller — APPROVED** [review]

**Pull request merged**